### PR TITLE
Fixing the moon activity

### PR DESCRIPTION
**Pull request by ricknjacky**
The issue #867 finds a resolution to it, in this pull request

### CODE REVIEW

**Comment by llaske** [comment]
The activity no longer work, the screen is blank and there is the following error in the console:

image

Look like neither `humane.js` neither `libnotify.css` are included in your PR.

File: activities/Moon.activity/js/moon-activity.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 define(['activity/data-model', 'activity/draw', 'webL10n', 'sugar-web/env', 'sugar-web/datastore', 'moment-with-locales.min'], function(DataModel, Draw, l10n, env, datastore, moment) {
 
     'use strict';
-
+    requirejs(['humane'], function (doc, humane) {
```
Comment (llaske) [inline activities/Moon.activity/js/moon-activity.js:4]: That's not correct, `humane` should be added in the `define` method above.

**Review by llaske — CHANGES_REQUESTED** [review]
Some issues to fix.
Please ensure the activity works on your side.

File: activities/Moon.activity/js/moon-activity.js [hunk 2]
```diff
@@ -319,7 +319,8 @@ define(['activity/data-model', 'activity/draw', 'webL10n', 'sugar-web/env', 'sug
 			file_size: 0
 		};
 		datastore.create(metadata, function() {
-			console.log("export done.")
+            humane.log(webL10n.get('MoonImageSaved'));
```
Comment (llaske) [inline activities/Moon.activity/js/moon-activity.js:322]: `webL10n` is not define in this context. I guess you should use `l10n`

Comment (ricknjacky) [reply activities/Moon.activity/js/moon-activity.js:322]: I did the changes the activity just doesn't load up now :( 
I am struggling to find the bug, please help me out here

File: activities/Moon.activity/js/moon-activity.js (hunk 1 above)
Comment (ricknjacky) [reply activities/Moon.activity/js/moon-activity.js:4]: sorry for that, did the change no difference

**Comment by ricknjacky** [comment]
Result after I did the requested changes.

image

**Comment by prakashujjwal1010** [comment]
- firstly you need to add `humane.js` ([here](https://github.com/llaske/sugarizer/blob/master/activities/QRCode.activity/lib/humane.js)) in `lib` directory to use it.
- then you need to add `libnotify.css` ([here](https://github.com/llaske/sugarizer/blob/master/activities/QRCode.activity/css/libnotify.css)) in `css` directory 
- add the `humane` in `define` method like `define([........, 'humane'], function(......., humane)`  instead of `requirejs` to

this would solve the problem. test it at your end.

**Pull request closed without merging**